plugins/inputs/nftables/nftables.go

```diff
@@ -0,0 +1,185 @@
+//go:generate ../../../tools/readme_config_includer/generator
+//go:build linux
+
+package nftables
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os/exec"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+)
+
+const measurement = "nftables"
+
+type Nftables struct {
+	UseSudo bool   `toml:"use_sudo"`
+	Binary  string `toml:"binary"`
+	Tables  []string
+}
+
+var nftableConfig = `
```

Comment:
Should be sampleConfig (unexported) following iptables pattern, or use embedded file like iptables does:
  //go:embed sample.conf
  var sampleConfig string